modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web/src/test/java/com/liferay/portal/search/tuning/synonyms/web/internal/portlet/action/EditSynonymSetsMVCActionCommandTest.java

```diff
@@ -90,22 +88,18 @@ public void testDoProcessAction() throws Exception {
 	}
 
 	@Test
-	public void testGetSynonymSetOptional() {
-		Optional<SynonymSet> synonymSetOptional =
-			_editSynonymSetsMVCActionCommand.getSynonymSetOptional(
-				Mockito.mock(SynonymSetIndexName.class), _actionRequest);
+	public void testGetSynonymSet() {
+		SynonymSet synonymSet = _editSynonymSetsMVCActionCommand.getSynonymSet(
```

Comment:
@LisztLiszt Please inline this one. Declare the variable in line 101, as we need to call methods on that one.